src/System.Text.Json/src/System/Text/Json/JsonConstants.cs

```diff
@@ -66,6 +66,6 @@ internal static class JsonConstants
         public const int HighSurrogateEndValue = 0xDBFF;
         public const int LowSurrogateStartValue = 0xDC00;
         public const int LowSurrogateEndValue = 0xDFFF;
-        public const int ShiftRightBy10 = 0x400;
+        public const int BitShiftBy10 = 0x400;
```

Comment:
I assume that it was being used for division (shift right) previously but now is also used for multiplication (shift left)?

Reply:
Yes.